doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -716,16 +716,12 @@ Libraries for networking
     * Support for using pre-provisioned X.509 certificates.
     * Support for using DTLS Connection Identifier
 
-  * Updated:
-
-    * Zephyr's LwM2M Connectivity Monitor object now uses a 16-bit value for radio signal strength so it does not roll over on values smaller than -126 dBm.
+  * Updated Zephyr's LwM2M Connectivity Monitor object to use a 16-bit value for radio signal strength so it does not roll over on values smaller than -126 dBm.
```

Comment:
```suggestion
  * Updated Zephyr's LwM2M Connectivity Monitor object to use a 16-bit value for radio signal strength so that it does not roll over on values smaller than -126 dBm.
```